spec/support/database_cleaner.rb

```diff
@@ -1,16 +1,6 @@
 RSpec.configure do |config|
-  config.before(:suite) do
+  config.before(:each) do
     DatabaseCleaner.clean_with(:truncation)
     Rails.application.load_seed
```

Comment:
> But you always expect the seed to be there.

Exactly the problem... each test should start with an empty database, and then only get filled with the necessary data needed by that test..